apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -154,6 +154,10 @@ const EVENTS = {
   SUMMARY_PAGE_NEXT_LEVEL_CLICKED: 'Summary Page Next Level Clicked',
   SUMMARY_PAGE_BACK_TO_LEVEL_CLICKED: 'Summary Page Back To Level Clicked',
 
+  // Check for understanding
+  CFU_NAMES_TOGGLED_ON: 'Summary Page Names Toggled On',
```

Comment:
I appreciate you adding these as you go. I forgot and will have to add them in afterwards :(

Reply:
And thank you for clarifying with Bryan which we need.